### ISSUE TITLE
Can I create common classes in shared projects that can be utilized by many other models? In other words can I setup remote dependencies to other projects/repos and branches?

### ISSUE THREAD

**Issue by agitana** [finos/legend#327]

**Comment by beekemarie** [comment]
We do support this feature internally but didn’t push it externally yet. In order for this to work we need to finish the SDLC build pipeline which is currently scheduled for end Q1. We will notify everyone when this feature is supported externally.

**Comment by AJ73** [comment]
Is this now supported @beekemarie  & @agitana ?

**Comment by beekemarie** [comment]
Hi @AJ73! Quick update on this, we are running slightly behind on the SDLC build pipeline but that work should be completed in the next 2-3 weeks. However, we realized that we cannot fully support this use case without open sourcing a first version of the metadata server as well.  We're still discussing prioritization and the timeline internally. Once we finalized the roadmap for this, I'll share more details with you.
